common/environment/configure/hardening.sh

```diff
@@ -28,3 +28,5 @@ else
 	FFLAGS="-fno-PIE ${FFLAGS}"
 	LDFLAGS="-no-pie ${LDFLAGS}"
 fi
+
+CFLAGS="${CFLAGS} -Werror=implicit"
```

Comment:
I worry that this may break autotools, specifically, `AC_CHECK_LIBS` or `AC_SEARCH_LIBS`

Reply:
Argh, true. I will test some packages that use it.